src/main/kotlin/commands/BanCommand.kt

```diff
@@ -0,0 +1,34 @@
+package commands
+
+import Command
+import ConfigManager.readConfigSafe
+import UserConfig
+import long
+import integer
+import string
+import doesLater
+import arg
+
+object BanCommand : Command("ec") {
+    init {
+        long("user") {
+            integer("deletemessagedays") {
+                string("reason") {
```

Comment:
Reason should have a user configurable default, so you don't need to require the reason